cg/cli/workflow/commands.py

```diff
@@ -26,7 +26,9 @@
 from cg.models.cg_config import CGConfig
 from cg.store.store import Store
 
-OPTION_YES = click.option("-y", "--yes", is_flag=True, help="Skip confirmation")
+OPTION_SKIP_CONFIRMATION = click.option(
+    "-y", "--skip-confirmation", "--yes", is_flag=True, help="Skip confirmation"
+)
```

Comment:
why are we creation an `OPTION_SKIP_CONFIRMATION` here? Is it not possible to import the `SKIP_CONFIRMATION` one that already exists here?

Reply:
Absolutely, we were already importing `DRY_RUN` and `FORCE` from `cli_options` there, so it makes perfect sense to also import `SKIP_CONFIRMATION`.